proto/einride/netlify/cms/v1/annotations.proto

```diff
@@ -242,6 +244,16 @@ message Widget {
   }
 }
 
+// The boolean widget translates a toggle switch input to a true/false value.
+message CustomWidget {
+  // Type of the custom widget.
+  string widget = 1;
+  // Widget options, as YAML keys and values provided to the custom widget.
+  map<string, string> options = 2;
+  // Widget options, as raw YAML strings appended to the custom widget.
+  repeated string raw_options = 3;
```

Comment:
Why we have two ways to config options? Would one way be enough or is there some reason we need both a map and a string list?

Reply:
The `options` variant is a bit more structured. It can express most options, but not all valid YAML. Shall we go with only `raw_options`?

Reply:
@liufuyang Updated!